include/deal.II/dofs/dof_tools.h

```diff
@@ -2385,6 +2385,27 @@ namespace DoFTools
    * @image html support_point_dofs1.png
    * @image html support_point_dofs2.png
    * </p>
+   *
+   * To generate the mesh and the support point info in a single gnuplot file,
+   * use code similar to
+   * @code
+   * std::ofstream out("gnuplot.gpl");
+   * out << "plot '-' using 1:2 with lines, '-' with labels point pt 2 offset 1,1" << std::endl;
+   * GridOut().write_gnuplot (triangulation, out);
+   * out << "e" << std::endl;
+   *
+   * std::map<types::global_dof_index, Point<dim> > support_points;
+   * DoFTools::map_dofs_to_support_points (MappingQ1<dim>(),
+   *                                       dof_handler,
+   *                                       support_points);
+   * DoFTools::write_gnuplot_dof_support_point_info(out,
+   *                                                support_points);
+   * out << "e" << std::endl;
+   * @endcode
+   * and plot using
```

Comment:
I suspect that this is a command to be entered inside gnuplot? Maybe say so.

Reply:
alternative is `gnuplot gnuplot.gpl` which IMO is more straight-forward.

Reply:
No, that opens and closes a window right away. You would then need a `pause -1` statement, but in any case you can't use `replot` then after changing some plotting parameters.

Reply:
> No, that opens and closes a window right away.

I did not look at the code, but personally I would prefer an example which writes into an .eps file as opposed to just plot it within the Gnuplot. I think that would be more common usage.